recipes/novelrt/all/conanfile.py

```diff
@@ -0,0 +1,154 @@
+from conans import ConanFile, CMake, tools, errors
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class NovelRTConan(ConanFile):
+    name = "novelrt"
+    version = "0.0.1"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/novelrt/NovelRT"
+    description = "A cross-platform 2D game engine accompanied by a strong toolset for visual novels."
+    topics = {"conan", "game", "engine" "gamedev", "visualnovel", "vn"}
+    settings = "os", "compiler", "build_type", "arch"
+    tool_requires = ("cmake/3.22.0")
+    requires = [
+        ("freetype/2.10.1"),
+        ("glfw/3.3.2"),
+        ("glm/0.9.9.7"),
+        ("gtest/1.10.0"),
+        ("libsndfile/1.0.30"),
+        ("ms-gsl/3.1.0"),
+        ("openal/1.21.1"),
+        ("onetbb/2021.3.0"),
+        ("spdlog/1.8.2"),
+        ("vulkan-loader/1.2.182"),
+        ("vulkan-memory-allocator/2.3.0")
+    ]
+    generators = "cmake_find_package", "cmake_paths"
+    default_options = {
+        "freetype:shared": True,
+        "glfw:shared": True,
+        "libsndfile:shared": True,
+        "openal:shared": True,
+        "PNG:shared": True,
+        "BZip2:shared": True,
+        "flac:shared": True,
+        "fmt:shared": True,
+        "onetbb:tbbproxy": False,
+        "onetbb:tbbmalloc": False,
+        "Opus:shared": True,
+        "Ogg:shared": True,
+        "Vorbis:shared": True,
+        "spdlog:header_only": True,
+        "vulkan-loader:shared": True,
+    }
+    cmake = None
+    unsupportedOS = ['AIX', 'Android', 'Arduino', 'Emscripten', 'FreeBSD', 'Macos', 'Neutrino', 'SunOS', 'WindowsCE', 'WindowsStore', 'iOS', 'tvOS', 'watchOS']
+    unsupportedCompilers = ['gcc', 'intel', 'mcst-lcc', 'qcc', 'sun-cc', 'msvc']
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "clang": "10",
+            "Visual Studio": "16"
+        }
+
+    @property
+    def _source_subfolder(self):
+        return "source"
+
+    @property
+    def _build_subfolder(self):
+        return "artifacts"
+
+    @property
+    def _is_static_lib_win(self):
+        return str(self.settings.compiler.runtime) in ["MTd", "MT"]
+
+    @property
+    def _is_debug_runtime_win(self):
+        return str(self.settings.compiler.runtime) in ["MDd", "MTd"]
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True)
+
+    def configure_cmake(self):
+        cmakePath = self.deps_env_info["cmake"].CMAKE_ROOT
+        if cmakePath:
+            if self.settings.os == "Windows":
+                cmakePath += "\\bin\\cmake.exe"
```

Comment:
shouldn't be necessary to manipulate with `cmake` paths here. conan should be able to automatically apply the right executable from build requirements.